### DISCUSSION TITLE
Syncing with fork of Davx5

### DISCUSSION THREAD

**Discussion by tysino** [TechbeeAT/jtxBoard#120] [Q&A] [answered]
Hello.
I am using a fork of DavX5 to sync my calendar (specifically, [this one](https://gitlab.e.foundation/e/os/AccountManager)). In this app I can select jtx board as the task app, but no new collections show up in jtx board. Of course, when I click "Sync with DavX5" in jtx board it does not detect DavX5 because the fork uses a different package name. Should this work? Or is there a restriction to the correct package name for DavX5?

Edit: Here are links to this issue on the /e/ project's [forum](https://community.e.foundation/t/accountmanager-jtx-board-support/46419) and their [GitLab](https://gitlab.e.foundation/e/backlog/-/issues/6508).

**Comment by patrickunterwegs** [comment, marked as answer]
Hey @tysino , so far I haven't been dealing with the /e/ project, so I investigated a bit. They seem to have changed a few things in their fork, but specifically they don't include the SyncAdapter for jtx Board. I'm not sure what specific strategy they follow with their approach. But basically as the SyncAdapter for jtx Board is not defined, it wouldn't work. 
As you already stated correctly, jtx Board does not see the fork due to the different package name. The collections would only appear after the first (successful) sync. There is no other dedicated restriction on the DAVx5 package name in jtx Board, but the problem here is really that the fork does not contain all the code that would make the jtx Board sync work... 

Maybe you could open a ticket for this issue on the /e/ project?

**Reply by tysino** [reply]
Wow, thanks for the elaborate answer! So this is a bug on the adapter side, because there I can select jtx board and would therefore expect compatibility.

**Reply by patrickunterwegs** [reply]
Yes, indeed, it seems like they have taken the code and UI elements, but not the sync adapter definition! If you open an issue on their side, it would be great if you could link it here 🙂

**Reply by unalignedcoder** [reply]
> Wow, thanks for the elaborate answer! So this is a bug on the adapter side, because there I can select jtx board and would therefore expect compatibility.

May I ask, what is the point of that fork? There is no explanation on their repository. Why is it supposed to be better than Davx?

**Reply by patrickunterwegs** [reply]
> > Wow, thanks for the elaborate answer! So this is a bug on the adapter side, because there I can select jtx board and would therefore expect compatibility.
> 
> May I ask, what is the point of that fork? There is no explanation on their repository. Why is it supposed to be better than Davx?

Well, I guess that's something only the /e/ project can answer 😉 from the code it looked like they try to have a solution that looks more like it's embedded in the OS...

**Reply by tysino** [reply]
> > Wow, thanks for the elaborate answer! So this is a bug on the adapter side, because there I can select jtx board and would therefore expect compatibility.
> 
> May I ask, what is the point of that fork? There is no explanation on their repository. Why is it supposed to be better than Davx?

They want to provide a full ecosystem as a replacement for google and the like. Therefore they need some account manager integrated in the OS (to provide e.g. settings synchronization). They fork a bunch of apps to test the whole system together and avoid surprises when the original apps get updates.
That said, I'm not a fan of this practice either, since it obviously has some drawbacks. And, since it is non-profit, I guess they could use their developer time elsewhere...
